02-microarray/clustering_microarray_01_heatmap.Rmd

```diff
@@ -137,9 +269,91 @@ heatmap
 dev.off()
 ```
 
-Print session info:
+Now, let's add some annotation bars to our heatmap.
+
+## Prepare metadata for annotation
 
-```{r Print session info}
-# Print session info 
+From the accompanying [paper](https://pubmed.ncbi.nlm.nih.gov/21107323/), we know that three PLX4032-sensitive parental cell lines (M229, M238 and M249) and three derived PLX4032-resistant (r) sub-lines (M229_r5, M238_r1, and M249_r4) were treated or not treated with the RAF-selective inhibitor, PLX4032 [@Nazarian2010].
+We are going to annotate our heatmap with the variables that hold the `refinebio_cell_line` and `refinebio_treatment` data.
```

Comment:
I think this paragraph should be updated to mention that we are going to create a new column from our existing metadata.